docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,217 @@
+[[Galleon_Provisioning]]
+= Provisioning WildFly with Galleon
+
+With the WildFly server being internally built using the https://github.com/wildfly/galleon[Galleon project], WildFly benefits +
+from Galleon's provisioning capabilities. One of the main features that Galleon brings to WildFly is the ability
+to install a complete or customized WildFly server using Galleon tooling.
+
+More information on Galleon features and tooling can be found in the Galleon https://docs.wildfly.org/galleon/[docs].
+
+== WildFly Galleon feature-packs
+
+WildFly is packaged as a Galleon feature-pack maven artifact (a zipped file that contains installation content). 
+This feature-pack, as well as the feature-packs on which its depends (core and servlet feature-packs),
+are deployed in public maven repositories. 
+
+Galleon is used to install WildFly, WildFly feature-packs are retrieved and 
+their content is assembled to create an installation.
+
+== Installing WildFly using Galleon
+
+The Galleon maven plugin or Galleon CLI are used to install WildFly. The latest CLI and documentation can be downloaded 
+from the Galleon releases https://github.com/wildfly/galleon/releases[page].
+
+To install the latest final version of WildFly into the directory _my-wildfly-server_ call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to run a complete WildFly server. 
+By default, all standalone and domain configurations are installed.
+
+[NOTE]
+====
+
+* Append the release version to install an identified release, for example:
```

Comment:
https://github.com/wildfly/wildfly/blob/0e0705acccc5485b6ed56fa3235b4c22e5a090cf/docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc shows what this "Note" bloc looks like.

I think either the Note bloc can be dropped, and its contents are just part of the main bloc or the '*' char at the start of L37, L42, L46 can be removed.  Having the bullet points in there doesn't look right to me as there's an example between each bullet.